### PR TITLE
Styled next up display

### PR DESCRIPTION
**Pull request by madikarizma**
### Changes proposed in this pull request:

* Added CSS for Next Up display
* Modified JS to accommodate UI needs
* Fixed Next Up container hover/click bug

Fixes #
JW7-2511

### CODE REVIEW

File: src/js/view/components/nextuptooltip.js [hunk 1]
```diff
@@ -114,14 +109,19 @@ define([
...
+              dom.removeClass(this.img, 'jw-nextup-thumbnail-visible');
+            }
 
             // set header
             this.header = element.getElementsByClassName('jw-nextup-header')[0];
             this.header.innerText = this._nextUpText;
             // set title
             this.title = element.getElementsByClassName('jw-nextup-title')[0];
-            this.title.innerText = nextUpItem.title || 'title';
+            this.title.innerText = nextUpItem.title || 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
```
Comment (johnBartos) [inline src/js/view/components/nextuptooltip.js]: Lore ipsum as a fallback - is this intentional?

Comment (egreaves) [inline src/js/view/components/nextuptooltip.js]: @johnBartos @madikarizma It's ok to see the behavior for now as this is a PR to the feature branch, not against feeds/master. I'll remove it before creating a PR for the ticket.

File: src/js/view/components/nextuptooltip.js [hunk 2]
```diff
@@ -57,11 +52,11 @@ define([
         },
         show : function(el) {
             el = el || this.container;
-            el.style.display = '';
+            dom.addClass(el, 'jw-nextup-container-visible');
```
Comment (johnBartos) [inline src/js/view/components/nextuptooltip.js]: There's a `toggleClass` function in utils

File: src/js/view/components/nextuptooltip.js [hunk 3]
```diff
@@ -114,14 +109,19 @@ define([
                 .on('click tap', this.playNext, this);
             // setup thumbnail
             this.img = element.getElementsByClassName('jw-nextup-thumbnail')[0];
-            this.image(this.loadThumbnail(nextUpItem.image));
+            if (nextUpItem.image) {
+              this.image(this.loadThumbnail(nextUpItem.image));
```
Comment (johnBartos) [inline src/js/view/components/nextuptooltip.js]: toggle

File: src/js/view/components/nextuptooltip.js (hunk 1 above)
Comment (madikarizma) [inline src/js/view/components/nextuptooltip.js]: @johnBartos @egreaves My bad. Meant to remove that, but as Evol said.. maybe keep it until after demo.

File: src/js/view/components/nextuptooltip.js (hunk 3 above)
Comment (madikarizma) [inline src/js/view/components/nextuptooltip.js]: would toggle work in a condition?

File: src/js/view/components/nextuptooltip.js (hunk 2 above)
Comment (madikarizma) [inline src/js/view/components/nextuptooltip.js]: my thoughts were only that add and remove are more obvious what the intent is

**Pull request merged**